src/components/Secrets/utils/__tests__/service-account-utils.spec.ts

```diff
@@ -36,17 +44,46 @@ const testComponents = [
 const testComponentsNames = [testComponent.metadata.name, 'test-component-2'];
 
 const testSecrets = [
-  imagePullSecret,
+  imagePullSecretWithNamespace,
   {
-    ...imagePullSecret,
-    metadata: { ...imagePullSecret.metadata, name: 'test-secret-2' },
+    ...imagePullSecretWithNamespace,
+    metadata: { ...imagePullSecretWithNamespace.metadata, name: 'test-secret-2' },
   },
 ] as SecretKind[];
 
 const k8sPatchResourceMock = createK8sUtilMock('K8sQueryPatchResource');
 const k8sGetResourceMock = createK8sUtilMock('K8sGetResource');
 const K8sListResourceItemsMock = createK8sUtilMock('K8sListResourceItems');
 
+describe('isLinkableSecret', () => {
+  it('should return true for dockerjson/dockercfg/basicAuth', () => {
+    for (const validType of [
+      SecretType.dockercfg,
+      SecretType.dockerconfigjson,
+      SecretType.basicAuth,
+    ]) {
+      const validSecret = { ...imagePullSecret, type: validType };
+      expect(isLinkableSecret(validSecret)).toBe(true);
+    }
+  });
+
+  it('should return false for ', () => {
```

Comment:
nit: update the label here to be something like "should return false for invalid secret types"? :)

Reply:
updated.